src/services/types.ts

```diff
@@ -485,6 +504,7 @@ namespace ts {
         getProgram(): Program | undefined;
 
         /* @internal */ getNonBoundSourceFile(fileName: string): SourceFile;
+        /* @internal */ getAutoImportProvider(ensureSynchronized?: boolean): Program | undefined;

```

Comment:
I dont think you want ensureSynchronized parameter at all here

Reply:
You’re right, this is old.